pkg/rslog/logger_impl.go

```diff
@@ -218,11 +219,24 @@ func UpdateDefaultLogger(options LoggerOptionsImpl, outputBuilder OutputBuilder)
 // to support switching to a legacy logger.
 // TODO: Remove this when all applications have fully moved to the new logging standard.
 func ReplaceDefaultLogger(logger Logger) {
+	// Doing this to avoid the ensureDefaultLoggerReadLock
+	// to overwrite the logger setted here
+	once.Do(func() {})
+
 	mutex.Lock()
 	defer mutex.Unlock()
 	defaultLogger = logger
 }
 
+// UseTerminalLogger sets the DefaultLoggerFactory variable with a TerminalLoggerFactory instance.
+// It makes the default logger have its output to STDERR with enhanced text formatting.
+// Make sure to call it before any call to DefaultLogger or package logging function
+// because the logger instance will be created for the first time that the DefaultLogger function
+// is called.
+func UseTerminalLogger(l LogLevel) {
+	DefaultLoggerFactory = &TerminalLoggerFactory{l}
+}
```

Comment:
Considering this will only be used by CLI programs, I think this is a fine solution. :smiley: